build.gradle

```diff
@@ -7,10 +7,10 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:1.5.0'
-        classpath 'com.google.gms:google-services:1.4.0-beta1'
+        classpath 'com.google.gms:google-services:1.3.1'
```

Comment:
I didn't update this. There is no practical difference for us. No need to use the beta dependency.